diff --git a/menu/menu_displaylist.c b/menu/menu_displaylist.c
index 315b9a89ef..1b942861d5 100644
--- a/menu/menu_displaylist.c
+++ b/menu/menu_displaylist.c
@@ -7153,9 +7153,7 @@ unsigned menu_displaylist_build_list(
                {MENU_ENUM_LABEL_MENU_DRIVER,           PARSE_ONLY_STRING_OPTIONS},
                {MENU_ENUM_LABEL_RECORD_DRIVER,         PARSE_ONLY_STRING_OPTIONS},
                {MENU_ENUM_LABEL_MIDI_DRIVER,           PARSE_ONLY_STRING_OPTIONS},
-#ifdef HAVE_LAKKA
                {MENU_ENUM_LABEL_WIFI_DRIVER,           PARSE_ONLY_STRING_OPTIONS},
-#endif
             };
 
             for (i = 0; i < ARRAY_SIZE(build_list); i++)
