diff --git a/configuration.c b/configuration.c
index fcd8fb4519..17479cba7e 100644
--- a/configuration.c
+++ b/configuration.c
@@ -559,11 +559,7 @@ static enum camera_driver_enum CAMERA_DEFAULT_DRIVER = CAMERA_ANDROID;
 static enum camera_driver_enum CAMERA_DEFAULT_DRIVER = CAMERA_NULL;
 #endif
 
-#if defined(HAVE_LAKKA)
 static enum wifi_driver_enum WIFI_DEFAULT_DRIVER = WIFI_CONNMANCTL;
-#else
-static enum wifi_driver_enum WIFI_DEFAULT_DRIVER = WIFI_NULL;
-#endif
 
 #if defined(ANDROID)
 static enum location_driver_enum LOCATION_DEFAULT_DRIVER = LOCATION_ANDROID;
diff --git a/menu/menu_setting.c b/menu/menu_setting.c
index 5bebf9364b..87aa2f3816 100644
--- a/menu/menu_setting.c
+++ b/menu/menu_setting.c
@@ -8290,7 +8290,6 @@ static bool setting_append_list(
          SETTINGS_DATA_LIST_CURRENT_ADD_FLAGS(list, list_info, SD_FLAG_ADVANCED);
 #endif
 
-#ifdef HAVE_LAKKA
          if (string_is_not_equal(settings->arrays.wifi_driver, "null"))
          {
             CONFIG_ACTION(
@@ -8301,7 +8300,6 @@ static bool setting_append_list(
                   &subgroup_info,
                   parent_group);
          }
-#endif
 
          CONFIG_ACTION(
                list, list_info,
