From 4330ddbbb219e4f8fbce89aaa6ef5d61f4ea1646 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bart=C5=82omiej=20Burdukiewicz?=
 <bartlomiej.burdukiewicz@gmail.com>
Date: Tue, 5 Jan 2021 13:45:15 +0100
Subject: [PATCH] Do not override default LDFLAGS.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Bartłomiej Burdukiewicz <bartlomiej.burdukiewicz@gmail.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 10e5c34..b2e6278 100644
--- a/Makefile
+++ b/Makefile
@@ -36,15 +36,15 @@ SOURCES := \
 ifneq ($(ISWIN),)
   OUTNAME := dosbox_pure_libretro.dll
   CXX     ?= g++
-  LDFLAGS :=  -Wl,--gc-sections
+  LDFLAGS +=  -Wl,--gc-sections
 else ifneq ($(ISMAC),)
   OUTNAME := dosbox_pure_libretro.dylib
   CXX     ?= clang++
-  LDFLAGS :=  -Wl,-dead_strip
+  LDFLAGS +=  -Wl,-dead_strip
 else
   OUTNAME := dosbox_pure_libretro.so
   CXX     ?= g++
-  LDFLAGS :=  -Wl,--gc-sections
+  LDFLAGS +=  -Wl,--gc-sections
 endif
 
 ifneq ($(and $(filter ARMv7,$(PROCCPU)),$(filter neon,$(PROCCPU))),)
