From 14c8097bcebe5f97f4490788ee982cb6df1e97e4 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Fri, 2 Apr 2021 13:29:08 +0200
Subject: [PATCH] workaround use_wayland

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/common/gl/context.cpp | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/src/common/gl/context.cpp b/src/common/gl/context.cpp
index 745639ba..5d1fa124 100644
--- a/src/common/gl/context.cpp
+++ b/src/common/gl/context.cpp
@@ -16,25 +16,8 @@ Log_SetChannel(GL::Context);
 #endif
 
 #ifdef USE_EGL
-#if defined(USE_WAYLAND) || defined(USE_GBM) || defined(USE_FBDEV) || defined(USE_X11)
-#if defined(USE_WAYLAND)
 #include "context_egl_wayland.h"
 #endif
-#if defined(USE_GBM)
-#include "context_egl_gbm.h"
-#endif
-#if defined(USE_FBDEV)
-#include "context_egl_fbdev.h"
-#endif
-#if defined(USE_X11)
-#include "context_egl_x11.h"
-#endif
-#elif defined(ANDROID)
-#include "context_egl_android.h"
-#else
-#error Unknown EGL platform
-#endif
-#endif
 
 #ifdef USE_GLX
 #include "context_glx.h"
-- 
2.25.1

